python_modules/libraries/dagstermill/dagstermill/examples/repository.py

```diff
@@ -355,14 +355,14 @@ def filepicklelist_resource(init_context):
             resource_defs={
                 "list": ResourceDefinition(lambda _: []),
                 "io_manager": fs_io_manager,
-                "file_manager": local_file_manager,
+                "output_notebook_io_manager": local_output_notebook_io_manager,
```

Comment:
Could `fs_io_manager` be used here for the `output_notebook_io_manager` as well?  If so, would it make sense for the default IO manager key for the notebook output to just be "io_manager"?

I don't have an opinion here either way, just exploring the options.

Reply:
no because 1) `fs_io_manager` pickles the content so users can't open the persisted file right away 2) `fs_io_manager` doesn't come with the `get_output_asset_key` so it won't yield asset materialization or provide handy asset key customization 

`OutputNotebookIOManager` base class kinda shield the asset key handling from notebook users - when they customize their own io manager by extending the class, they don't have to worry about "how do i get the notebook show up in asset catalog."